libtcmu_common.h

```diff
@@ -39,6 +39,9 @@ struct tcmulib_cmd;
 #define CFGFS_ROOT "/sys/kernel/config/target"
 #define CFGFS_CORE CFGFS_ROOT"/core"
 
+#define VPD_MAX_UNMAP_LBA_COUNT            1048576
```

Comment:
Assuming the block size is 512.

Currently I just set the count to 1MB and the max size will be 1MB * 512 = 512MB, which will be larger than 32MB.

From the "VMware vSphere Storage APIs – Array Integration (VAAI).pdf", we can know: the Data Mover is asked to move data in terms of 32MB transfers, and there has not limitation for the UNMAP. So just set size max limitation in UNMAP command to 512MB.

But this is not correct, actually this should depend on device's max_hw_segment_size (here is the size of the object for ceph).